test/deals/post-deals.api-test.ts

```diff
@@ -0,0 +1,78 @@
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('POST /deals', () => {
+  const valueGenerator = new RandomValueGenerator();
+
+  const portfolioIdentifier = 'E1';
```

Comment:
Constant please.

Reply:
Sorry I don't understand what the desired state of the code is -- this was just intended as test data, like `expectedDealIdentifierResponse` below. It's not considered a re-usable constant outside of this test class?

Reply:
`E1` will be repeated across various tests, if in future this constant value needs to be amended (very unlikely though) it will be nice to refer to these values from constants please?